### ISSUE TITLE
[BUG] AD Stuck leaving a field, working with CP

### ISSUE THREAD

**Issue by Chico0008** [Stephan-S/FS19_AutoDrive#1563]
**AutoDrive Version:** 1.1.0.5-rc5 
**Release or Custom Zip:** Release  
**Map (If applicable):** Fenton Forest  
**Vehicle / Type (If applicable):**   Tractor with Loader
**Multiplayer or Singleplayer:** Singleplayer  

**Describe the bug**  
I got 2 field working with CP to load Grass/Straw (working with tractor + loader)
CP is configured to use AD for loading/unloading only
I already use this with previous version en CP and AD, and it worked fine, but since the last version of AD, the tractor got stuck when trying to leave the field 
See capture and log attached, messages at 2020-08-04 13:45

**If applicable - Steps to reproduce the behavior**  
make a field work to load grass/hay/Sraw with CP, 
make CP use AD to unload

**Screenshots**  
![1](https://user-images.githubusercontent.com/58886243/89290535-4c5fc680-d659-11ea-9947-9408edc2539b.png)
![2](https://user-images.githubusercontent.com/58886243/89290543-51bd1100-d659-11ea-8361-995fb48230a5.png)

[log.zip](https://github.com/Stephan-S/FS19_AutoDrive/files/5021884/log.zip)

**Comment by Axel32019** [comment]
Hello,
Champ 1 is the destination for this field, correct?
You should not activate CP in AD each time, see button 21 https://github.com/Stephan-S/FS19_AutoDrive/wiki/Buttons-with-short-description
This is only to be activated when AD is on the way to the field. If field work has started CP and AD handle everthing automatic by themselve!
For the situation of your screenshot:
If done these points, what happen if you deactivate AD, drive back 2-3 CP waypoints, start CP at the closest point?

**Comment by Chico0008** [comment]
Even without the button 21 activated, it's the same

i go back 3/4 point, start again CP, stop, and then same thing happen when AD activate, he stop because it doesn't seems to find a way out of the field

**Comment by SteelRodent** [comment]
I've just run into something similar trying to get AD to refill slurry for CP. 
When CP hands over to AD (when the slurry spreader is empty), AD drives to the deliver point and then gets stuck.

On first attempt AD failed to follow the network route and instead slowly crawled across the field until it gave up and said "can't calculate a path"; and then released the vehicle, but CP was still in control, only nothing happened (think that's how it's supposed to work when AD fails a run for CP).

On second attempt I switched exit method to "nearest point" (from default). This time AD drove to the deliver point and shut off the engine, and then just sat there doing nothing, but still in control of the tractor. AD still says "Drive to loading point", only it's not.

I did dry runs to make sure either point is reachable, but on a live test it just fails. This setup used to work fine.
This is with CP 061 and AD 1.1.0.5-RC5
